### PR TITLE
docs/usdts: add info on targeting all shared libraries for a process

### PR DESCRIPTION
**Pull request by jordalgo**
Just a docs update.

##### Checklist

- [ ] Language changes are updated in `man/adoc/bpftrace.adoc` and if needed in `docs/reference_guide.md`
- [ ] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

File: docs/reference_guide.md [hunk 1]
```diff
@@ -2990,7 +2990,7 @@ Return true if the string haystack contains the string needle, and zero otherwis
 Examples:
 
 ```
-bpftrace -e 't:syscalls:sys_enter_execve /strcontains(str(args.filename),"bin")/ { @[comm, str(args.filename)] = count(); }'
```
Comment (viktormalik) [inline docs/reference_guide.md:2993]: Don't update lines not relevant for the PR please.

File: docs/reference_guide.md [hunk 2]
```diff
@@ -1359,7 +1359,7 @@ usdt:library_path:probe_name
 usdt:library_path:[probe_namespace]:probe_name
 ```
 
-Where `probe_namespace` is optional if `probe_name` is unique within the binary.
+Where `probe_namespace` is optional if `probe_name` is unique within the binary. You can also target all shared libraries and the main executable in a process's address space if you use the `*` wildcard in place of the `binary_path`/`library_path` AND pass in a pid command line argument e.g. `bpftrace -e 'usdt:*:loop { printf("hi\n"); }' -p 1234`.
```
Comment (viktormalik) [inline docs/reference_guide.md:1362]: This applies to uprobes, too, doesn't it? Shouldn't it be placed elsewhere, then?

Comment (jordalgo) [reply docs/reference_guide.md:1362]: Good point. How about in the probes intro section or perhaps I should add a new "-p" item in the Usage section that points this out.

Comment (viktormalik) [reply docs/reference_guide.md:1362]: Either *Probes* intro or a new subsection of *Probes* seems the best place to me. I'm also wondering if we should reflect this in manpage, too (we should really put some work to unify these two).

Comment (jordalgo) [reply docs/reference_guide.md:1362]: Actually, this method of targeting doesn't work for uprobes/uretprobes. Anyway, going to abandon in favor of [this PR](https://github.com/iovisor/bpftrace/pull/2734)

**Pull request closed without merging**